### PR TITLE
feat(explorer): replace Sia Central with explored API for the host route (minus benchmarks)

### PR DESCRIPTION
**Pull request by telestrial**
The only snaffu here is that the benchmark aspect of host is not ready yet, so I simply commented it out for now. I'll spawn a task for it.

### CODE REVIEW

**Comment by telestrial** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/SiaFoundation/web/819?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#819** <a href="https://app.graphite.dev/github/pr/SiaFoundation/web/819?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/SiaFoundation/web/819?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#818** <a href="https://app.graphite.dev/github/pr/SiaFoundation/web/818?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: apps/explorer/components/Host/HostPricing.tsx [hunk 1]
```diff
@@ -10,48 +11,52 @@ import { SiaCentralExchangeRates } from '@siafoundation/sia-central-types'
 import { useMemo } from 'react'
 import {
   getDownloadCost,
-  getDownloadSpeed,
-  getRemainingOverTotalStorage,
-  getRemainingStorage,
+  // getDownloadSpeed,
```
Comment (alexfreska) [inline apps/explorer/components/Host/HostPricing.tsx:14]: Lets make sure to have GitHub issues for each feature we need to add back in.

**Review by alexfreska — APPROVED** [review]
lgtm, did we previously add a basic test?

**Comment by telestrial** [comment]
### Merge activity

* **Nov 21, 12:47 PM EST**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/SiaFoundation/web/819).

**Comment by telestrial** [comment]
> lgtm, did we previously add a basic test?

We have e2e tests for all routes on the Explorer now, including host, if that's what you're asking.

**Comment by telestrial** [comment]
Closes https://github.com/SiaFoundation/web/pull/819

**Pull request merged**